layouts/pricing/index.html

```diff
@@ -0,0 +1,499 @@
+               <sup>$</sup>105 &#47; GB memory per month
+             </p>
+             <p>
+               Non-production systems
+             </p>
+             <p>
+                An environment for each of your teams
+             </p>
+             <p>
+               Non-production domain names
+             </p>
+             <!-- <p>
+               Full access to Add-on (<i class="fa fa-fw fa-plus-square" aria-hidden="true"></i>) Services
+             </p> -->
+             <p>
+               <a href="mailto:cloud-gov-inquiries@gsa.gov?subject=Purchasing%20Prototyping%20Package" class="button">Contact us</a><br/>
+               <a href="/overview/pricing/prototyping/" class="small-link">About Prototyping packages &rarr;</a>
+             </p>
+           </li>
+           <li>
+             <h2 class="production">Production</h2>
+             <p>
+               Go to production with an accelerated path to ATO.
+             </p>
+             <p>
+               Monthly fee starts at:
+               <span><strong><sup>$</sup>1667</strong> &#47; FISMA Low system</span><br />
+               <span><strong><sup>$</sup>7500</strong> &#47; FISMA Moderate system</span>
+             </p>
+             <p>
+               <sup>$</sup>105 &#47; GB memory per month
+             </p>
+             <p>
+               One system
+             </p>
+             <p>
+               Multiple environments (e.g. dev, stage, prod) supporting one system
+             </p>
+             <p>
+               Support for your agency domain name
+             </p>
+             <!-- <p>
+               Full access to Add-on (<i class="fa fa-fw fa-plus-square" aria-hidden="true"></i>) Services
+             </p> -->
+             <p>
+               <a href="mailto:cloud-gov-inquiries@gsa.gov?subject=Purchasing%20FISMA%20Package" class="button">Contact us</a><br/>
+               <a href="/overview/pricing/rates/#fisma-low" class="small-link">About FISMA packages &rarr;</a>
+             </p>
+           </li>
+         </ul>
+      </section>
+
+      <section class="usa-grid">
+        <h2>All use of cloud.gov includes:</h2>
+        <!-- <p>
+          Placeholder text.
+        </p> -->
+        <div class="usa-width-one-half">
+          <ul class="column-single">
+            <li>
+              FedRAMP JAB-authorized hosting
+            </li>
+            <li>
+              Self-service provisioning of services
+            </li>
+            <li>
+              Self-service deployment of apps
+            </li>
+            <li>
+              Basic technical and compliance support (during business hours)
+            </li>
+            <li>
+              Documentation/tutorial material
+            </li>
+            <li>
+              Continuous platform-level scanning and security updates
+            </li>
+          </ul>
+        </div>
+
+        <div class="usa-width-one-half">
+          <p>
+            Access to free services
```

Comment:
We should move `Access to free services` to the bottom of the column to the left, and just make this text `Free services`.